libraries/AP_SpdHgtControl/AP_SpdHgtControl_Heli.cpp

```diff
@@ -0,0 +1,182 @@
+#include "AP_SpdHgtControl_Heli.h"
+#include <AP_HAL/AP_HAL.h>
+
+extern const AP_HAL::HAL& hal;
+
+// table of user settable parameters
+const AP_Param::GroupInfo AP_SpdHgtControl_Heli::var_info[] = {
+
```

Comment:
Add an enable parameter here.  in the description add "DO NOT SET MANUALLY.  Parameter set based on autorotation enable."

Reply:
Done